README.md

```diff
@@ -1,70 +1,194 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
 
 ### Prerequisites
-HTML & CSS
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
 ### Setup
-git clone <repocitory URL>
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
 ### Install
 
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
 ### Usage
 
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
 ### Run tests
 
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
 ### Deployment
 
+You can deploy this project using:
 
+<!--
+Example:
 
-## Authors
+```sh
 
-👤 **Author1**
+```
+ -->
 
-- GitHub: [@githubhandle](https://github.com/dselasea)
-- Twitter: [@twitterhandle](https://twitter.com/danielsanyidoho)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/danielsanyidoho)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+<!-- AUTHORS -->
 
-Contributions, issues, and feature requests are welcome!
+## 👥 Authors <a name="authors"></a>
 
-Feel free to check the [issues page](../../issues/).
+> Mention all of the collaborators of this project.
 
-## Show your support
+👤 **Author1**
 
-Give a ⭐️ if you like this project!
+- GitHub: [@dselasea](https://github.com/dselasea)
+- Twitter: [@danielsanyidoho](https://twitter.com/danielsanyidoho)
+- LinkedIn: [Daniel Selase Anyidoho](https://www.linkedin.com/in/danielsanyidoho/)
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+<!-- LICENSE -->
 
-## 📝 License
+## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
\ No newline at end of file
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Please make sure to include a LICENSE to your project according to the previous reviewer, at the moment this is not working. Microverse suggests you use the MIT one. Follow the [official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this! 🤓

Public repositories on GitHub are often used to share open-source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software.